tools/c7n_azure/tests_azure/tests_resources/test_cosmos_db.py

```diff
@@ -13,6 +13,22 @@
 from c7n.utils import local_session
 
 
+def get_ext_ip():
+    # local external ip needs to be added to the database when recording
+    from requests import get
+    return get('https://api.ipify.org').text
```

Comment:
if we really do need it, then this one might be more inline to not depending on a new third party https://checkip.amazonaws.com/